wrapper/pre_copy.py

```diff
@@ -698,6 +715,36 @@ def _get_nbdkit_cmd(self, disk, filters, final):
 
         return nbdkit_cmd
 
+    def _get_nbdkit_cmd_ssh(self, disk, filters, final):
+        print("*** %s" % disk.absolute_path.replace('.vmdk', '-flat.vmdk'))
+        nbdkit_cmd = [
+            'nbdkit',
+            '-v',
+            '-U', disk.sock,
+            '-P', disk.pidfile,
+            '--exit-with-parent',
+            '--readonly',
+            '--foreground',
+            '--exportname=/',
+        ] + [
+            '--filter=' + f for f in filters
+        ] + [
+            '--filter=log',
+            'ssh',
+            # pylint: disable=protected-access
```

Comment:
Accidental leftover?

Reply:
It was already there. And if I remove it, tox tells me ```Access to a protected member _moId of a client class (protected-access)```, because we access the `_moId` attribute of the VM.